src/js/shot-generator/Editor.js

```diff
@@ -194,6 +195,23 @@ const SceneManager = connect(
 
     let cameraHelper = useRef(null)
 
+    const createPosePresetFromBones = (preset) =>  {
```

Comment:
For now, I'd recommend we remove the `createPosePresetFromBones` function, and remove any newly added references to `createPosePreset` that aren't required. To avoid confusion about what is/isn't being currently used.